Gems/EMotionFX/Code/EMotionFX/Source/DualQuatSkinDeformer.cpp

```diff
@@ -324,6 +310,8 @@ namespace EMotionFX
         SkinningInfoVertexAttributeLayer* skinningLayer = (SkinningInfoVertexAttributeLayer*)m_mesh->FindSharedVertexAttributeLayer(SkinningInfoVertexAttributeLayer::TYPE_ID);
         MCORE_ASSERT(skinningLayer);
 
+        AZStd::vector<uint16> localBoneMap(highestJointId + 1, AZStd::numeric_limits<uint16>::max());
```

Comment:
ultra nit: I would do a const uint16 invalidBoneIndex = AZStd::numeric_limits<uint16>::max() before line 313, then compare boneIndex to invalidBoneIndex in line 331.
Define a invalid bone index somewhere else would be great too.